Formula/openssl.rb

```diff
@@ -67,58 +63,18 @@ def install
     dirs = []
 
     archs.each do |arch|
```

Comment:
It remains redundant, but the above block also needs tweaking, since `each` implies more than one which will no longer be possible. That's what I meant to say, more clearly 😉. Not sure if the retention of building on 32-bit CPUs was intentional or not.